servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,28 @@
+---
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Certain features come with a cost and in some cases the cost of having a feature active is not worth the performance impact on the system.
+
+For example, in a development environment you will want a lot of logging information to support problem analysis. For many users the [Debug Session](/servicecontrol/plugins/debug-session.md) plugin is really useful at this stage of the application life-cycle but you would never put that plugin into production. For the same reason, Particular advise that customers not use the [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugin outside of development.
```

Comment:
this is really 3 examples bundle together. perhaps one intro phrase and point form?